packages/wonder-blocks-testing/src/respond-with.js

```diff
@@ -196,7 +219,18 @@ export const makeMockResponse = (
                 );
             });
 
+        /* istanbul ignore next */
         default:
-            throw new Error(`Unknown response type: ${response.type}`);
+            if (process.env.NODE_ENV !== "production") {
+                // If we're not in production, give an immediate signal that the
+                // dev forgot to support this new type.
+                throw new Error(`Unknown response type: ${response.type}`);
+            }
```

Comment:
**note:** I did some checking when writing this to see what this all compiles down to. And although our testing library is likely only used in test, I thought it useful to still keep things useful in the various usage paths.

For folks that didn't know, like me, turns out that:

1. Such checks compile to `if (true) { /* the code */ }` or `if (false) { /*no code*/ }`; I expected the whole condition to be removed if the value was `false`.
2. If code is not in an `if` like this, it gets included regardless.

Not a big deal for this library, but worth noting when doing this elsewhere. In addition, this affects the Wonder Blocks build that we package; not the final usage point (i.e. webapp gets the "production" version, even in dev and testing).